src/components/ids-multiselect/ids-multiselect.scss

```
@@ -0,0 +1,42 @@
+@import '../../core/ids-base';
+
+::slotted(.options){
+  border-top: 1px solid #B7B7BA;
+  padding-top: 4px;
+}
+
+::slotted(.is-selected){
+  @include bg-azure-20();
+  @include text-black();
+}
+
+ids-trigger-field{
```

```suggestion
ids-trigger-field {
```

Lint might complain about all of these?

---

sure I can change it are you seeing errors on your end, my lint hasn't had any issues with this one

---

Ok thought it was - but please, prefer the space for consistency